src/llm/apis/openai_completions.cpp

```diff
@@ -782,6 +782,93 @@ std::string OpenAIChatCompletionsHandler::serializeUnaryResponse(const std::vect
     return buffer.GetString();
 }
 
+std::string OpenAIChatCompletionsHandler::serializeUnaryResponse(const ov::genai::EncodedResults& results) {  // TODO separate common part with function implemented above
```

Comment:
Is it really a thing, that we have code duplication because non-continuous batching pipelines return different ov::genai objects as result?

If so, isnt there an API that could return us the same type as for continuous batching?
Is it due to the fact that it lacks logprobs? Only because of that?

It will make our code very hard to maintain, since all changes need to be made in multiple places